vendor/k8s.io/kubernetes/pkg/apiserver/audit/audit.go

```diff
@@ -82,18 +84,23 @@ var _ http.Hijacker = &fancyResponseWriterDelegator{}
 func WithAudit(handler http.Handler, attributeGetter apiserver.RequestAttributeGetter, out io.Writer) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
 		attribs := attributeGetter.GetAttribs(req)
-		asuser := req.Header.Get("Impersonate-User")
+		asuser := req.Header.Get(authenticationapi.ImpersonateUserHeader)
 		if len(asuser) == 0 {
 			asuser = "<self>"
 		}
+		asgroups := "<lookup>"
+		requestedGroups := req.Header[authenticationapi.ImpersonateGroupHeader]
+		if len(requestedGroups) > 0 {
+			asgroups = strings.Join(requestedGroups, ", ")
```

Comment:
Updated to something like this: `"'group 1', 'group 2'"`, according to @deads2k upstream comment.